requirements/dev.txt

```diff
@@ -30,8 +30,8 @@ ordereddict==1.1
 # If you are updating this, you might want to update
 # the hashes of the hooks repositories listed in the
 # .pre-commit-config.yaml file as well.
-# sha256: AnUshRna3Tz1V6c2KD9KB3LF72xU6XwRPs3_1mtnrKw
-pre-commit==0.5.5
+# sha256: wapVuGj_cYauuuwMQDlL2MHHiGdXK3J-WpxzS56oL9I
+pre-commit==0.6.2
```

Comment:
There are no newer releases in the pre-commit-hooks (https://github.com/pre-commit/pre-commit-hooks) repository. So we are good.